<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>735</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>735</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>381</width>
     <height>87</height>
    </rect>
   </property>
   <property name="plainText">
    <string>	            ЛИЧНЫЙ КАБИНЕТ
Здесь Вы можете зашифровать или расшифровать Ваш текст</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="CryptInput">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>651</width>
     <height>271</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Введите Ваш текст</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>520</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>651</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="CryptButton">
      <property name="text">
       <string>Зашифровать текст</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="DecryptButton">
      <property name="text">
       <string>Расшифровать текст</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="KeyI">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>520</y>
     <width>330</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Введите ключ для расшифровки</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
